man/systemd-cryptenroll.xml

```diff
@@ -401,6 +402,41 @@
         <xi:include href="version-info.xml" xpointer="v248"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--fido2-salt-file=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>When enrolling a FIDO2 security token, specifies the path to a file or an
+        <constant>AF_UNIX</constant> socket from which we should read the salt value to be used in the
+        HMAC operation performed by the FIDO2 security token. If this option is not specified, the salt
+        will be randomly generated.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--fido2-salt-file-offset=<replaceable>BYTES</replaceable></option></term>
```

Comment:
hm, what's the rationale for having this? I'd really not clutter the interface with options we don't need. i mean, i know we have this for unlock keyfiles, but i don't like it much either there. But for fido2 stuff i see even less of a usecase.

Reply:
Agreed, I've only added that to match the systemd-cryptsetup, but since you're fine with this option missing, I've removed it.